wiki/Styling-concept.md

```diff
@@ -0,0 +1,99 @@
+
+### CSS vs. CSS modules
+
+In React the style of "normal" CSS files like _Example.css_ are defined globally. Therfore you don't need to explicitly import the CSS file to use the style. Thus be very careful when using normal CSS files and keep in mind that the style you define can be used in any file of the repository.
+For example when you define the following style...
+
+```
+.button {
+    background-color: red;
+}
+```
+
+... this might lead to confusion because whenever someone uses the class button now this style is applied no matter if it was intended or not.
+
+If you want to apply style just to spefific files and not globally react has a solution called CSS modules. Instead of creating a file _Example.css_ you have to create _Example.module.css_. This file you have to explicitly import in every file you want to use it in. For example like this:
+
+```
+import styles from './Example.module.css';
+```
+
+Now let's continue with this example. Let's say in _Example.module.css_ we have defined the following because we just want the buttons of this file to be green:
+
+```
+.button {
+    background-color: green;
+}
+```
+
+In the file we would include the style in the following way:
+
+```
+<button className={styles.button}></button>
+```
+
+### Conventions
+
+- naming:
+  For the naming of classes and ids please use **hyphens** consistently.
+  For example don't call the class `buttonBackground` and instead call it `button-background`.
+- sizing:
+  Try to only use relative units (vw,vh,rem,%) to size elements and **not** absolut units (px)
+- coloring:
+  Try to only use the predefined css variables of our color schema when coloring a component/element.
+
+---
+
+## Corporate / Visual Identity
+
+### Colors:
+
+- color-primary: #00C2FF
+- color-primary-inverted: #1C272B
+- color-primary-inverted-2: #2F3C41
+- color-primary-inverted-text: #FFFFFF
+- color-background: #EFEFEF
+- color-background-2: #FFFFFF
+- color-background-text: #1D1D1F
+- color-cta: #FF6B00
+
+How to use this color schema:
+
+- color-primary: Is the primary color of our brand. It is used for important elements (e.g headlines)
+- color-primary-inverted: Is the main color complementing the primary color. For example it can be used for the header/footer.
+- color-primary-inverted-2: Is the another color complementing the primary color. It is useful in combination with color-primary-inverted.
+- color-primary-inverted-text: Is the color of all text written on color-primary-inverted or color-primary-inverted-2.
+- color-background: Is the main background color of the website and should therefore be used for the coloring of the background.
+- color-background-2: Is another background color that can be used on the main background color (e.g. for containers).
+- color-background-text: The color of all the text that is either written on color-background or color-background-2
+- color-cta: Is the "call-to-action" color and thus is used for elements like buttons.
```

Comment:
Can you maybe add a color table like this:
![grafik](https://user-images.githubusercontent.com/38314662/107491675-c5e9eb80-6b8b-11eb-9e8d-481c27e8a367.png)
Should be possible with simple HTML. Helps as overview

Reply:
done